Registry/registry.var

```diff
@@ -264,10 +264,9 @@ rconfig   integer   report_end              namelist,wrfvar5  1  10000000 - "rep
 rconfig   integer   tovs_start              namelist,wrfvar5  1  1        - "tovs_start"               ""  ""
 rconfig   integer   tovs_end                namelist,wrfvar5  1  10000000 - "tovs_end"                 ""  ""
 rconfig   logical   gpsref_thinning         namelist,wrfvar5  1  .false.  - "gpsref_thinning"          ""  ""
+rconfig   logical   outer_loop_restart      namelist,wrfvar6  1  .false.  - "outer_loop_restart"       ""  ""
 rconfig   integer   max_ext_its             namelist,wrfvar6  1  1        - "max_ext_its"              ""  ""
 rconfig   integer   ntmax                   namelist,wrfvar6  max_outer_iterations  75      - "ntmax"                    ""  ""
-rconfig   logical   use_inverse_squarerootb namelist,wrfvar6  1  .false.  - "use_inverse_squarerootb"       ""  ""
-rconfig   logical   use_interpolate_cvt     namelist,wrfvar6  1  .false.  - "use_interpolate_cvt"       ""  ""
```

Comment:
You must use an older version of code here. These three lines of namelist change need to be recovered to the latest 'develop' branch code.

Reply:
Address review comments.